pcli/src/command/tx.rs

```diff
@@ -162,6 +165,24 @@ pub enum TxCmd {
     /// Currently, only zero-fee sweep transactions are implemented.
     #[clap(display_order = 990)]
     Sweep,
+
+    /// Perform an ICS-20 withdrawal.
+    #[clap(display_order = 1000)]
+    Ics20Withdrawal {
+        destination_chain_id: String,
+        destination_chain_address: String,
+        denom: String, //TODO: should we pull this out of amount
+        amount: String,
+        source_channel: String,
+
+        #[clap(long, default_value = "0", display_order = 100)]
+        timeout_height: u64,
+        #[clap(long, default_value = "0", display_order = 150)]
+        timeout_timestamp: u64,
+
+        #[clap(long, default_value = "0", display_order = 200)]
+        return_address_index: u32,
```

Comment:
This seems like it should be a `source` flag, like the other commands (e.g., `tx send`), and to match those, we should have it (1) always select funds only from the given source account and (2) always choose a return address that's a randomization of the given source account.

This way, we uphold the design that the pcli commands respect accounts as an (internal) accounting boundary.